content/rc/administration/setup/change-plan.md

```diff
@@ -7,22 +7,45 @@ categories: ["RC"]
 aliases: /rc/administration/setup-and-editing/changing-subscription-plan/
         /rc/administration/setup-and-editing/changing-subscription-plan/
 ---
-It is very easy to upgrade or downgrade your subscription at any time,
-all with no downtime.
+You can change your subscription plan at any time.
 
-To do this:
+To do so:
 
-1. Go to "Subscriptions" in the menu
-1. Click on the subscription name you want to edit
-1. Click on "Change Subscription" button at the bottom of the form
-1. Click on which plan you would like to upgrade or downgrade to
-1. Click the "Update" button
+1. From the Redis Cloud menu, choose **Subscriptions**.  
 
-Be aware that you cannot downgrade your plan if the databases in your
-subscription are larger in total size than the next lowest plan. For
-example, if you are currently on the 2.5GB plan and your databases total
-1.75GB, you would not be able to downgrade to the 1GB plan as your
-databases currently exceed the plan's limit. If you need to downgrade,
-you must remove data or databases to get below the threshold.
+    ![Select **Subscriptions** from the Redis Cloud menu](/images/rc/subscription-menu-select.png)
 
-{{< youtube uCcCnQ74C2U >}}
+1.  Select the subscription you want to change.
+
+    ![Select the target subscription from the list](/images/rc/subscription-list-select.png)
+
+1.  Select the **Change Description** button to list the available options.
+
+    ![The **Change subscription** button appears below the list of subscriptions.](/images/rc/subscription-change-button.png)
+
+
+1.  The first option you can change is type, which indicates whether the plan supports replication or data-persistence.  
```

Comment:
Do you want to make "type" bold or italics, or highlight it in some way?

Reply:
Not in this case, as the control is not named in the UX and there is no item labelled "type."

Reply:
From your image it looks like the options are either Cache or Standard. Maybe we can point that out? Something like and let them know which is which? Maybe something like:
You can choose between two types of plans. "Cache" supports replication and data-persistence, while "Standard" does not. Your current selection is highlighted and you can select your preferred option to change this setting.

Reply:
It's a good idea; however, I'd like to separate that from this work item and make it part of a new work item devoted to UX descriptions.